src/openrct2/paint/Paint.cpp

```diff
@@ -863,8 +863,8 @@ paint_struct* PaintAddImageAsChild(
     int32_t bound_box_length_y, int32_t bound_box_length_z, int32_t z_offset, int32_t bound_box_offset_x,
     int32_t bound_box_offset_y, int32_t bound_box_offset_z)
 {
-    assert(bound_box_length_x > 0);
-    assert(bound_box_length_y > 0);
+    assert(bound_box_length_x >= 0);
+    assert(bound_box_length_y >= 0);
```

Comment:
The asserts are not required anymore. Can be removed.

Reply:
Ok, I’ll remove them. Btw, do you happen to know why there were introduced?